EnableJupyterDisplay.swift

```diff
@@ -0,0 +1,178 @@
+#if canImport(Cryptor)
+import Cryptor
+#endif
+
+import Foundation
+
+enum JupyterDisplay {
+    struct Header: Codable {
+        let msgId: String
+        let username: String
+        let session: String
+        var date: String {
+            let currentDate = Date()
+            let formatter = DateFormatter()
+            formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSZZZZZ"
+            formatter.timeZone = TimeZone(secondsFromGMT: 0)
+            formatter.locale = Locale(identifier: "en_US_POSIX")
+            return formatter.string(from: currentDate)
+        }
+        let msgType: String
+        let version: String
+
+        public init(
+            msgId: String = UUID().uuidString,
+            username: String = "kernel",
+            session: String,
+            msgType: String = "display_data",
+            version: String = "5.2"
+            ) {
+            self.msgId = msgId
+            self.username = username
+            self.session = session
+            self.msgType = msgType
+            self.version = version
+        }
+
+        public func toJSON() -> String {
+            do {
+                let encoder = JSONEncoder()
+                encoder.outputFormatting = [.prettyPrinted, .sortedKeys]
+                encoder.keyEncodingStrategy = .convertToSnakeCase
+                let jsonData = try encoder.encode(self)
+                let jsonString = String(data: jsonData, encoding: .utf8)!
+                return jsonString
+            }
+            catch {
+                return "{}"
+            }
+        }
+    }
+
+    struct Message {
+        var msgType: String = "display_data"
+        var delimiter = "<IDS|MSG>"
+        var key: String = ""
+        var header: Header
+        var metaData: String = "{}"
```

Comment:
"Metadata" is a word, so I'd suggest calling this `metadata` instead of `metaData`.